server.js

```diff
@@ -24,10 +16,58 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints = /shows & /titles/title & year/year & cast/cast & genre/listed_in & country/country&type=Movie or TV Show')
```

Comment:
You can create an HTML file and render it here to make it look nicer, or use the template engine like ejs: https://ejs.co/